K8s-deployment/K8s-cluster/addons/mon-stack/README.md

```diff
@@ -1,6 +1,24 @@
-## Pre-Requisites
-Secrets needed 
+# Installation
+
+## Generate secret files
+1. Make a copy of sample secrets directory.
+```
+cp -r example-secrets/secrets .
```

Comment:
`example-secrets` not present in the repository.
Can add a command in the secret creation script itself to create the `secrets` directory. And remove this step altogether.